src/main/java/org/broadinstitute/consent/http/resources/ConsentResource.java

```diff
@@ -70,9 +87,6 @@ public Response describeInvalidConsents() {
     public Response createConsent(@Context UriInfo info, Consent rec, @Auth User user) {
         try {
             DACUser dacUser = dacUserAPI.describeDACUserByEmail(user.getName());
-            if (rec.getTranslatedUseRestriction() == null) {
-                rec.setTranslatedUseRestriction(translateServiceAPI.translate(TranslateType.SAMPLESET.getValue(),rec.getUseRestriction()));
-            }
```

Comment:
This call, and the `update` below, are the real meat of this PR. We don't need to call out to ontology for this.